### PR TITLE
Chyam/10 restructure repo

### LINKED ISSUE #10: Restructure repo
Restructure repo according to audiences, generally by:
- how_to
- use_cases
- industry_solution_accelerator

### PR DESCRIPTION
**Pull request by ryubidragonfire**
Restructure to:
── AUTHORS.md
├── CODE_OF_CONDUCT.md
├── LICENSE
├── README.md
├── README_old.md
├── SECURITY.md
├── SUPPORT.md
├── archive
│   └── Labs
│       └── Clustering-and-Embedding
├── documentation
│   └── lease_extraction
├── fundamentals
│   ├── document_analysis
│   └── large_data
├── quick_start
├── requirements.txt
├── solution_accelarators
└── use_cases
    ├── call_center
    ├── generate_podcast_synopsis
    └── give_an_appropriate_name

Kept almost all content, except arhive/labs/clustering-and-embedding/ due to duplications. Welcome author to come forward for integration. 

This is not perfect, may require another iterations to refine.

### CODE REVIEW

**Comment by ryubidragonfire** [comment]
@jimmyliao @aldelar @aschauhan22 @louis-li @brandoncowenms I believe you have contributed to the repo. 

As per the v-team decision to reorganise the repo, I have given the first go. 

This is not perfect yet, however, it will servce as the first iteration. 

Please review:
- overall structure
- your specific contributions, to try to fit the overal format as much as possible. 
   - consider create a PR on this branch to work on your specific folder

Please add your name to AUTHOR.md. 

Please let contributors know about this if I have missed out any. 

If possible, please kindly review today, as I have only 1 hour Friday to address the feedback, and I will have less time next week to work on this repo. 

Thank you for your collaboration.

**Comment by aldelar** [comment]
Overall looks good @ryubidragonfire with some notes here:

1) Broken image link in the doc here: https://github.com/Azure/azure-openai-workshop/blob/chyam/10-restructure-repo/use_cases/give_an_appropriate_name/README.md

2) https://github.com/Azure/azure-openai-workshop/tree/chyam/10-restructure-repo/use_cases/give_an_appropriate_name

Should the content of that folder be split between 'quick starts' and then the rest moved under a new folder in 'fundamentals' called 'leveraging_langchain' or 'build_with_langchain' as a few are langchain related?

@louis-li feedback on this?

3) https://github.com/Azure/azure-openai-workshop/tree/chyam/10-restructure-repo/use_cases/call_center
Should move the .wav files to a 'data' folder in this location to be consistent with the rest of the repo.

4) https://github.com/Azure/azure-openai-workshop/tree/chyam/10-restructure-repo/solution_accelarators
directory name is misspelled: 'accelarators' instead of 'accelerators'

**Review by aldelar — CHANGES_REQUESTED** [review]
See my comments in the PR.
Would be nice to take care of these things if you can tomorrow before we merge this?

**Comment by louis-li** [comment]
Thanks Chew-Yean for your hard work. 

There are a few minor issues we can update later:
1. https://github.com/Azure/azure-openai-workshop/blob/chyam/10-restructure-repo/archive/Labs/Clustering-and-Embedding/Lab-clustering-embeddings-themes.ipynb

The image should be updated or removed - We cannot use OpenAI logo.

2. https://github.com/Azure/azure-openai-workshop/blob/chyam/10-restructure-repo/fundamentals/document_analysis/README.md

Some reference links are same as the ones in the READM.md in root folder

3. https://github.com/Azure/azure-openai-workshop/blob/chyam/10-restructure-repo/quick_start/README.md

The message should be updated. The Environmental variable section no longer apply.

**Comment by louis-li** [comment]
> Overall looks good @ryubidragonfire with some notes here:
> 
> 1. Broken image link in the doc here: https://github.com/Azure/azure-openai-workshop/blob/chyam/10-restructure-repo/use_cases/give_an_appropriate_name/README.md
> 2. https://github.com/Azure/azure-openai-workshop/tree/chyam/10-restructure-repo/use_cases/give_an_appropriate_name
> 
> Should the content of that folder be split between 'quick starts' and then the rest moved under a new folder in 'fundamentals' called 'leveraging_langchain' or 'build_with_langchain' as a few are langchain related?
> 
> @louis-li feedback on this?

In this folder, only first notebook is not langchain related. I like the idea of naming the folder 'work_with_langchain' or something similar. We can move the first notebook to another suitable place.

**Comment by ryubidragonfire** [comment]
> Overall looks good @ryubidragonfire with some notes here:
> 
> 1. Broken image link in the doc here: https://github.com/Azure/azure-openai-workshop/blob/chyam/10-restructure-repo/use_cases/give_an_appropriate_name/README.md [done]
> 2. https://github.com/Azure/azure-openai-workshop/tree/chyam/10-restructure-repo/use_cases/give_an_appropriate_name
> 
> Should the content of that folder be split between 'quick starts' and then the rest moved under a new folder in 'fundamentals' called 'leveraging_langchain' or 'build_with_langchain' as a few are langchain related? [done]
> 
> @louis-li feedback on this?
> 
> 3. https://github.com/Azure/azure-openai-workshop/tree/chyam/10-restructure-repo/use_cases/call_center
>    Should move the .wav files to a 'data' folder in this location to be consistent with the rest of the repo. [done]
> 4. https://github.com/Azure/azure-openai-workshop/tree/chyam/10-restructure-repo/solution_accelarators
>    directory name is misspelled: 'accelarators' instead of 'accelerators'  [done]

**Comment by ryubidragonfire** [comment]
click wrong button

**Comment by ryubidragonfire** [comment]
> > Overall looks good @ryubidragonfire with some notes here:
> > 
> > 1. Broken image link in the doc here: https://github.com/Azure/azure-openai-workshop/blob/chyam/10-restructure-repo/use_cases/give_an_appropriate_name/README.md
> > 2. https://github.com/Azure/azure-openai-workshop/tree/chyam/10-restructure-repo/use_cases/give_an_appropriate_name
> > 
> > Should the content of that folder be split between 'quick starts' and then the rest moved under a new folder in 'fundamentals' called 'leveraging_langchain' or 'build_with_langchain' as a few are langchain related?
> > @louis-li feedback on this?
> 
> In this folder, only first notebook is not langchain related. I like the idea of naming the folder 'work_with_langchain' or something similar. We can move the first notebook to another suitable place. [done]

**Comment by ryubidragonfire** [comment]
> See my comments in the PR. Would be nice to take care of these things if you can tomorrow before we merge this?

[done]

**Comment by ryubidragonfire** [comment]
Thank you for the comments. 
@louis-li @aldelar  please kindly re-review. I have a small window of time at night my time, if there are further comments before merge. I will have little time for the rest of the week.

File: README.md
Comment (louis-li) [inline README.md:1]: Can we add a section how to prepare environment for the code execution? Make it easier to use.

**Review by louis-li — APPROVED** [review]

**Review by aldelar — APPROVED** [review]
Thank you @ryubidragonfire for taking care of the suggestions. Looks good. We can easily tune the rest from this new and cleaner structure.

**Pull request merged**